### PR TITLE
Edits for consistency, rhetorical flow and concision

### PR DESCRIPTION
**Pull request by argonaut0**
These are my suggestions. May be a little bit heavy on the edits but makes the writing more clear in my opinion.

### CODE REVIEW

File: _posts/2024-02-09-flipper-zero.md [hunk 1]
```diff
@@ -12,72 +12,72 @@ With contributions from the rest of Maple Bacon
 
 On February 8 2024, the Ministry of Public Safety [released a statement][stmt] that, following the National Summit on Combatting Auto Theft, they intend to take various actions to discourage auto theft. This is in response to a large amount of auto theft, and there are [various][story1] stories [about cars being stolen in Canada and shipped overseas][story2] to be sold.
 
-As part of these actions, the Government of Canada wishes to ban certain hardware development tools like the Flipper Zero that have been used to steal some cars which have faulty security design. These tools have many legitimate uses such as security evaluation, understanding interoperability, and learning about wireless communication.
+As part of these actions, the Government of Canada wishes to ban certain hardware development tools like the Flipper Zero that have allegedly been used to steal cars with faulty security design. These tools have many legitimate uses such as security evaluation, understanding interoperability, and learning about wireless communication.
```
Comment (omentic) [inline _posts/2024-02-09-flipper-zero.md:15]: I like your changes a lot, but I like "have been used to steal some cars" over "have allegedly been used to steal cars"

**Review by omentic — APPROVED** [review]
i like these changes

File: _posts/2024-02-09-flipper-zero.md [hunk 2]
```diff
@@ -12,74 +12,74 @@ With contributions from the rest of Maple Bacon
...
 
-We cannot ban the chips either because they have a huge number of non-malicious uses and have innumerable alternatives, and doing so would significantly hurt Canadian industry, while doing very little to nothing to stop thieves getting them into the country. Commercial devices with the chips could equally be reprogrammed to send malicious data; there is no conceivable regulation scheme that would stop these from being used for stealing insecure cars.
+We cannot prevent people from building such devices by banning the chips either because they have a huge number of legitimate uses, and doing so would significantly hurt Canadian industry. Commercially availible consumer devices with the chips could equally be reprogrammed to send malicious data; there is no conceivable regulation scheme that would stop these from being used for stealing insecure cars.
 
-### It is impossible to craft a regulation to ban all software defined radio capable devices without impacting vital systems
+### It is Impossible to Craft a Regulation to Ban All Software Defined Radio Capable Devices without Impacting Vital Systems
```
Comment (omentic) [inline _posts/2024-02-09-flipper-zero.md:58]: without title case. also SDR-capable might be better

**Comment by omentic** [comment]
title case seems inconsistent & it might be better not in title case but everything else looks good

File: _posts/2024-02-09-flipper-zero.md (hunk 1 above)
Comment (argonaut0) [reply _posts/2024-02-09-flipper-zero.md:15]: I do as well, but I don't think flipper zeros have actually been used to steal any cars? as i understand its mostly amplification attacks thats the issue rn right? well I guess it says hardware tools *like* the flipper zero. yeah ok will change

**Comment by argonaut0** [comment]
Updated

**Pull request merged**